test/AutoRest.TestServer.Tests/Infrastructure/TestServerTestBase.cs

```diff
@@ -73,7 +73,7 @@ public Task TestStatus(Func<string, HttpPipeline, Task<Response>> test, bool ign
         private Task TestStatus(string scenario, Func<string, HttpPipeline, Task<Response>> test, bool ignoreScenario = false) => Test(scenario, async (host, pipeline) =>
         {
             var response = await test(host, pipeline);
-            Assert.That(response.Status, Is.EqualTo(200).Or.EqualTo(201), "Unexpected response " + response.ReasonPhrase);
+            Assert.That(response.Status, Is.EqualTo(200).Or.EqualTo(201).Or.EqualTo(202).Or.EqualTo(204), "Unexpected response " + response.ReasonPhrase);
```

Comment:
Allowed other accept values for tests that returned them. Pretty much anything in 200's should be good, but these tests only used 200, 201, 202, and 204.